tests/sentry/utils/sdk_crashes/test_event_stripper.py

```diff
@@ -1,335 +1,312 @@
-import abc
+import pytest
 
 from fixtures.sdk_crash_detection.crash_event import (
     get_crash_event,
     get_crash_event_with_frames,
     get_frames,
 )
-from sentry.db.models import NodeData
-from sentry.testutils import TestCase
-from sentry.testutils.cases import BaseTestCase
-from sentry.testutils.silo import region_silo_test
+from sentry.utils.pytest.fixtures import django_db_all
 from sentry.utils.safe import get_path, set_path
 from sentry.utils.sdk_crashes.cocoa_sdk_crash_detector import CocoaSDKCrashDetector
 from sentry.utils.sdk_crashes.event_stripper import strip_event_data
 
 
-class BaseEventStripperMixin(BaseTestCase, metaclass=abc.ABCMeta):
-    @abc.abstractmethod
-    def create_event(self, data, project_id, assert_no_errors=True):
-        pass
+@pytest.fixture
```

Comment:
The change is basically adding two fixture methods, removing the test mixing, converting all tests to functions, and adding `@django_db_all, @pytest.mark.snuba` to these tests. The functionality of the tests didn't change at all.

https://github.com/getsentry/sentry/blob/1ec94d6ee7c836dd0b9c00090b0512f19b18021d/tests/sentry/utils/sdk_crashes/test_event_stripper.py#L14-L50